src/Microsoft.AspNet.HttpOverrides/OverrideHeaderExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.HttpOverrides;
+
+namespace Microsoft.AspNet.Builder
+{
+    public static class OverrideHeaderExtensions
+    {
+        /// <summary>
+        /// Forwards proxied headers onto current request
+        /// </summary>
+        /// <param name="builder"></param>
+        /// <returns></returns>
+        public static IApplicationBuilder UseOverrideHeaders(this IApplicationBuilder builder)
```

Comment:
Remove this overload, it doesn't do anything because everything is turned off by default.